web-ui/src/main/java/com/tutrit/controller/UserController.java

```diff
@@ -0,0 +1,44 @@
+package com.tutrit.controller;
+
+
+import com.tutrit.bean.User;
+import com.tutrit.gateway.UserGateway;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.Optional;
+
+@Controller
+public class UserController {
+
+    @Autowired(required = false)
+    private UserGateway userGateway;
+
+    @GetMapping("users/{userId}")
+    public ModelAndView findUserById(@PathVariable String userId){
+        var mov = new ModelAndView();
+        mov.addObject("user",userGateway.findUserById(userId));
+        mov.setViewName("user-form.html");
```

Comment:
Did you test it? 
did it find the form, because it is shouldn't: there is no need for extension in a view name

Reply:
I created mock and the user-form worked with extension .html, but I removed anyway